azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/config/FunctionAppConfig.java

```diff
@@ -19,6 +19,8 @@ public class FunctionAppConfig extends AppServiceConfig {
     private boolean disableAppInsights;
     private String storageAccountName;
     private String storageAccountResourceGroup;
+    private String nativeExecutablePath;
```

Comment:
Please remove these new parameters as `FunctionAppConfig` is used to create resource in Azure, which should not contains native binary configurations